modules/acm/src/main/java/edu/gemini/epics/acm/CaCommandSender.java

```diff
@@ -103,16 +69,21 @@
      * @throws CaException
      */
     CaParameter<Integer> addInteger(String name, String channel,
-                                    String description) throws CaException;
+                                    String description, boolean isCADParameter) throws CaException;
 
-    CaParameter<Integer> addInteger(String name, String channel)
-            throws CaException;
+    default CaParameter<Integer> addInteger(String name, String channel)
+            throws CaException {
+        return addInteger(name, channel, null, true);
+    }
 
-    CaParameter<Integer> addInteger(String name, String channel,
-                                    String description, boolean isCADParameter) throws CaException;
+    default CaParameter<Integer> addInteger(String name, String channel, boolean isCADParameter) throws CaException {
+        return addInteger(name, channel, null, isCADParameter);
```

Comment:
Again, these `null` values make me kind of uncomfortable, especially when there is no documentation in the getter that states that the return value can be `null`.

Reply:
Now the documentation states the description can be `null`.